tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
can we remove this (as well as the new image under `public/`) please? or, if it was intentional, ideally that can be added in a separate PR

Reply:
It was intentional; I added the image to show the effect of the new options, but it didn't look quite like the existing one. I added the `tsconfig.json` for my own convenience in my IDE and because the Replit Vim and Emacs repos have them.

I've removed both. I can make new PRs for either or both, but happy to be guided by you.

Reply:
ahh I see, thanks. great point. would be great to add the tsconfig in a separate PR! as for the image, I think keeping the original is fine since that's the default.